### ISSUE TITLE
Tests to check if file readers & models work

### ISSUE THREAD

**Issue by guimarqu** [atoptima/ColunaDemos.jl#5]

**Comment by rafaelmartinelli** [comment]
FacilityLocation does not work. Even if you use Scanner, the instance file is not in the right format.

**Comment by guimarqu** [comment]
> FacilityLocation does not work. Even if you use Scanner, the instance file is not in the right format.

Indeed, I started to do it to try Benders decomposition and we switched to a single mode multi item lot sizing demo. 
I'm not even sure if the play file is a Facility location instance...
